task2/src/routes/index.ts

```diff
@@ -0,0 +1,46 @@
+import * as express from 'express';
+import crudService from '../crudService/crudSrvice';
+import userSchema from '../schema/userSchema';
+
+const userRouter = express.Router();
+
+userRouter.get('/', (req, res) => {
+  res.send(crudService.getAllUsers());
+});
+
+userRouter.get('/get/:id', (req, res) => {
+  const userFromDataset = crudService.getUser(req.params.id);
+  res.send(userFromDataset.length ? userFromDataset : 'No user found!');
+});
+
+userRouter.get('/getAuto/:id', (req, res) => {
+  const userSuggestions = crudService.getAutoSeggestions(req.params.id, 2);
```

Comment:
Better to get amount from request and set default param to 2.

Reply:
done, thanks